enterprise/views.py

```diff
@@ -160,6 +165,34 @@ def render_page_with_error_code_message(request, context_data, error_code, log_m
     )
 
 
+def get_create_ent_enrollment(
+        course_id,
+        enterprise_customer_user,
+):
+    """
+    Get or Create the Enterprise Course Enrollment.
+
+    This method will set the Source if the Enterprise Course Enrollment (ECE)
+    was already created. It is possible the ECE will have been created by the
+    background task at the time of enrollment so we need to update the source
+    in that case.
+    """
+    source = EnterpriseEnrollmentSource.get_source(EnterpriseEnrollmentSource.ENROLLMENT_URL)
+    # Create the Enterprise backend database records for this course
+    # enrollment
+    enterprise_course_enrollment, created = EnterpriseCourseEnrollment.objects.get_or_create(
```

Comment:
Just want to make sure I'm reading this correctly: if this call does create the enrollment, it will also set its source to _ENROLLMENT_URL_?

Reply:
Yes. If it `GETs` the Enrollment it will not change the Source.